man/tpm2_pcrlist.1.md

```diff
@@ -0,0 +1,72 @@
+tpm2_pcrlist 1 "AUGUST 2017" tpm2-tools
+==================================================
+
+NAME
+----
+
+tpm2_pcrlist(1) - List PCR values.
+
+SYNOPSIS
+--------
+
+`tpm2_pcrlist` [OPTIONS]
+
+DESCRIPTION
+-----------
+
+tpm2_pcrlist(1) Displays PCR values.
+
+OPTIONS
+-------
+
+  * `-g`, `--algorithm`=_HASH\_ALGORITHM_:
+    Only output PCR banks with the given algorithm.
```

Comment:
I guess you should also add a note here saying that section "Supported Hash Algorithms" lists the pretty names for algorithms or something in those lines. And also mention that all these algorithms may not be supported by the TPM and -s should be used to list the supported ones.

Reply:
Done